src/utils/download.ts

```diff
@@ -14,7 +14,7 @@
  *
  */
 import * as axios from 'axios';
-import * as fs from 'fs';
+import fs from 'fs';
```

Comment:
```suggestion
import * as fs from 'fs';
```
why are we changing this?

Reply:
unfortunately, stubs don't work when we create the alias for `fs` if you know any solution lets make the change.